### PR TITLE
added dayview and itinerary overview

### PR DESCRIPTION
**Pull request by shansingh14**
Finished up the DayView and ItineraryOverview pages and linked the backend as well.

### CODE REVIEW

File: packages/travel-project-frontend/src/pages/DayView/DayView.tsx [hunk 1]
```diff
@@ -1,83 +1,45 @@
...
 import Button from "@mui/material/Button";
 import Grid from "@mui/material/Grid";
 import Container from "@mui/material/Container";
 import Box from "@mui/material/Box";
+import { getEvents } from "../../httpClient/event";
+import { useEffect, useState } from "react";
 
 // Dummy Data for cards, change to fit DB Schema later
-interface Booking {
+interface Event {
```
Comment (calvinvanoss) [inline packages/travel-project-frontend/src/pages/DayView/DayView.tsx]: should be able to import type def directly from prisma.
import { Event } from "@prisma/client"

File: packages/travel-project-frontend/src/pages/ItineraryOverview/ItineraryOverview.tsx [hunk 2]
```diff
@@ -1,7 +1,108 @@
...
+  Card,
+  CardContent,
+  Typography,
+  Button,
+  Container,
+} from "@mui/material";
 
-function ItineraryOverview(): React.ReactElement {
-  return <div></div>;
+interface Day {
```
Comment (calvinvanoss) [inline packages/travel-project-frontend/src/pages/ItineraryOverview/ItineraryOverview.tsx]: can also import from prisma, see above

**Review by calvinvanoss — CHANGES_REQUESTED** [review]

**Review by calvinvanoss — APPROVED** [review]
looks good

**Review by seamus0192 — APPROVED** [review]
Looks good, will check this out for the specific routing stuff good stuff

**Comment by ScottHufschmidt** [comment]
No comments, looks great to me!

**Comment by shansingh14** [comment]
no revisions from me, looks good!

**Pull request merged**